toc.json

```diff
@@ -219,6 +219,17 @@
         }
       ]
     },
+    {
+      "type": "group",
+      "title": "SCIM",
```

Comment:
We should put SCIM into coming soon guide. Otherwise partner will assume its available in Prod.